xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -32,24 +37,20 @@ constexpr int ASS_BORDER_STYLE_SQUARE_BOX = 4; // Square box + outline
 
 constexpr int ASS_FONT_ENCODING_AUTO = -1;
 
-// Directory where user defined fonts are located (and where mkv fonts are extracted to)
+// Directory where user defined fonts are located (and where temporary MKV fonts are extracted to)
 constexpr const char* userFontPath = "special://home/media/Fonts/";
-// Directory where Kodi bundled fonts (default ones like Arial or Teletext) are located
+// Directory where Kodi bundled fonts files are located
 constexpr const char* systemFontPath = "special://xbmc/media/Fonts/";
 
-std::string GetDefaultFontPath(std::string& font)
+const char* GetSystemFontPath(const char* font)
```

Comment:
would be nice to keep all generic methods with `std::string` instead of `char *`. I'll do it later on the tmp cleanup if you don't want to get bothered with it